component/src/main/java/org/wso2/extension/siddhi/io/email/sink/EmailSink.java

```diff
@@ -127,11 +130,18 @@
                                 + " then addresses can be given as a comma separated list.",
                            type = DataType.STRING,
                            optional = true,
-                           defaultValue = "None")
-
+                           defaultValue = "None"),
+                @Parameter(name = "attachments",
+                        description = "File paths of the files that needs to be attached to the email.\n" +
+                                "These paths should be absolute paths.\n" +
+                                "They can be either a directories or files. If it's a directory all the files inside " +
```

Comment:
Should be "They can be either directories or files".

Reply:
fixed by 3d4e83b2f5d203578f81e8084b33d5770a6d0b18